Form-Controls/index.html

```diff
@@ -13,15 +13,61 @@
     <h1>Product Pick</h1>
   </header>
   <main>
+
   <form>
+
     <!-- write your html here-->
+    
+      <label for="firstname">First name:</label>
+      <input type="text" name="firstname" id="firstname" placeholder="Enter your name" required>
+<br>
+      <label for="lastname">Last name:</label>
+      <input type="text" name="lastname" id="lastname" placeholder="Enter your lastname" required>
+<br>
+      <label for="Email">Email:</label>
+      <input type="email" name="Email" id="Email" required placeholder="Enter your Email">
+<br>
+      <p>Select a colour:</p>
```

Comment:
```suggestion
      <p>Select a colour:</p>
```

When you ran LIghthouse, what did it say about this line? (Or the one after it?) It should give you some useful help on associating form controls with labels. 

https://dequeuniversity.com/rules/axe/4.0/label

....update. I just ran Lighthouse myself and it didn't pick this up for some reason. But please label these selects. Thank you!